code/modules/uplink/uplink_items.dm

```diff
@@ -1086,10 +1086,17 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
 
 /datum/uplink_item/device_tools/codespeak_manual
 	name = "Codespeak Manual"
-	desc = "Syndicate agents can be trained to use a series of codewords to convey complex information, which sounds like random concepts and drinks to anyone listening. This manual teaches you this Codespeak. You can also hit someone else with the manual in order to teach them. This is the deluxe edition, which has unlimited used."
-	item = /obj/item/codespeak_manual/unlimited
+	desc = "Unlimited use, read it and apply it to a friend's head! Anyone listening will think you're babbling about random concepts and drinks as you tell your friend to get ready, you're slipping the Captain, right now."
```

Comment:
Tried to condense this for the sake of pointing out it has unlimited use because that's more important to read first than last, especially when next to a cheaper one use any language book on the side. The fluff text also went on for quite some time.